lib/apiservers/engine/backends/convert/stats.go

```diff
@@ -42,6 +43,11 @@ type ContainerStats struct {
 	curDockerStat *types.StatsJSON
 	currentMetric *metrics.VMMetrics
 
+	// disk & net stats are accumulated during the life of the
+	// subscription. These maps will assist in that accumulation.
+	diskStats map[string]metrics.VirtualDisk
+	netStats  map[string]metrics.Network
```

Comment:
We drop the stats when the stats stream (subscription) ends.  That could be via the client or the API.  If the interaction is without streaming (`docker stats --no-stream`) then there's no accumulation and only a point in time look at disk and network.